### PR TITLE
Add `catppuccin`, `nightfox` and `tokyo-night` extensions

### PR DESCRIPTION
**Pull request by ssaunderss**
This adds the `catppuccin`, `nightfox` and `tokyo-night` themes extensions from https://github.com/ssaunderss/zed-themes. I bumped them into their own repos so they can be their own standalone extensions.

### CODE REVIEW

**Comment by maxdeviant** [comment]
@ssaunderss We actually do support reading multiple extensions out of a single repo.

So you could keep them in one repository, add it as a submodule, and then make the `path`s in `extensions.toml` point to subdirectories in that repository.

**Comment by maxdeviant** [comment]
> @ssaunderss We actually do support reading multiple extensions out of a single repo.
> 
> So you could keep them in one repository, add it as a submodule, and then make the `path`s in `extensions.toml` point to subdirectories in that repository.

Separate repos works too, so it all depends on if you'd rather keep them in a single repository or not 🙂

**Comment by ssaunderss** [comment]
@maxdeviant I think it'd be better to split them off for future development

File: extensions.toml [hunk 1]
```diff
@@ -13,3 +13,15 @@ version = "0.0.1"
 [vscode-dark-plus]
 path = "extensions/vscode-dark-plus"
 version = "0.0.1"
+
+[zed-catppuccin]
```
[the remark this answers is not in the document]
Comment (ssaunderss) [reply extensions.toml]: Done

**Comment by maxdeviant** [comment]
@ssaunderss `extensions.json` needs to be renamed to `extension.json` in the `zed-tokyo-night` and `zed-nightfox` repos:

https://github.com/ssaunderss/zed-tokyo-night/blob/48a0075ffb94ccaa4132c8e37f36fe69da7b614a/extensions.json

https://github.com/ssaunderss/zed-nightfox/blob/055bee3d93de806702e792837fe34b774ab6a602/extensions.json

**Comment by ssaunderss** [comment]
@maxdeviant Good catch, just updated them!

**Comment by maxdeviant** [comment]
@ssaunderss Thanks for making these themes available to all Zed users 🙂

**Pull request merged**